rpcd/playbooks/verify-maas.yml

```diff
@@ -23,7 +23,7 @@
         . {{ maas_venv_bin }}/activate
         {% endif %}
         {{ maas_rpc_scripts_dir }}/rpc-maas-tool.py verify-created \
-        --entity {{ inventory_hostname }} \
+        --entity {{ inventory_hostname }}{{ maas_fqdn_extension }} \
```

Comment:
verify-mass is still using inventory_hostname, which doesn't give solution that mix using hostname (ansible_hostname and inventory_name) in the same role (maas).